src/tmx/TmxUtils/src/SNMPClient.h

```diff
@@ -15,6 +15,15 @@ namespace tmx::utils
         GET,
         SET,
         OTHER // Processing this request type is not a defined behavior, included for testing only
+    }; 
+
+    struct snmpRequest {
+        std::string oid;
```

Comment:
Do we have a way to validate if oid is valid?

Reply:
No, we did not before either.